{"name": "test_pytest_rerunfailures[4-passed]", "status": "passed", "attachments": [{"name": "stdout", "source": "d87cb962-a30e-41e7-b7c6-3223f8e265d2-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "countdown", "value": "4"}, {"name": "status", "value": "'passed'"}], "start": 1710872971257, "stop": 1710872971283, "uuid": "3378467b-328f-4b4a-b8b2-81313aca833c", "historyId": "43246a5701389fee2e9f849d8c24f44e", "testCaseId": "df06b7d101485b30103206befbeb4d8a", "fullName": "tests.allure_pytest.externals.pytest_rerunfailures.pytest_rerunfailures_test#test_pytest_rerunfailures", "labels": [{"name": "feature", "value": "Integration"}, {"name": "parentSuite", "value": "tests.allure_pytest.externals.pytest_rerunfailures"}, {"name": "suite", "value": "pytest_rerunfailures_test"}, {"name": "host", "value": "MacBook-Pro-whenessel.local"}, {"name": "thread", "value": "29518-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.allure_pytest.externals.pytest_rerunfailures.pytest_rerunfailures_test"}], "links": [{"type": "issue", "url": "140", "name": "140"}]}